code/modules/projectiles/guns/mounted.dm

```diff
@@ -165,27 +165,33 @@
 
 	scatter = 10
 	deployed_scatter_change = -10
-	fire_delay = 0.5 SECONDS
+	fire_delay = 0.4 SECONDS
+	burst_amount = 12
+	burst_delay = 0.25 SECONDS
```

Comment:
could probs make this lower. The long delay between bursts means you get punished pretty bad just shooting wildly, and its a little slow.